src/bundle/Resources/public/scss/_card.scss

```diff
@@ -60,6 +60,28 @@
     }
 }
 
+.ez-label {
```

Comment:
If this is a separate component it should be in different file `_label.scss`.

Reply:
Ok.